test/unit_tests/modules/util/atomic-helpers-spec.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var assert = chai.assert;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1701/commits/6c4e6138b154fa642dab80dd973d720ca1690b5b